Block/SentryScript.php

```diff
@@ -48,9 +48,9 @@ public function __construct(
      */
     public function canUseScriptTag($blockName)
     {
-        return $this->dataHelper->isActive() &&
-            $this->dataHelper->useScriptTag() &&
-            $this->dataHelper->showScriptTagInThisBlock($blockName);
+        $jsTracking = ($this->dataHelper->useScriptTag() && $this->dataHelper->showScriptTagInThisBlock($blockName));
+
+        return $this->dataHelper->isActive() && ($jsTracking || $this->dataHelper->useSessionReplay());
```

Comment:
```suggestion
        if (!$this->dataHelper->isActive()) {
            return false;
        }
        
        if ($this->dataHelper->useScriptTag() && $this->dataHelper->showScriptTagInThisBlock($blockName)) {
            return true;
        }
        
        if ($this->dataHelper->useSessionReplay()) {
            return true;
        }
        
        return false;
```

This may help with readability. 
And i have another question about it, does it load the script twice this way? Do we need the `showScriptTagInThisBlock` check in this if as well, or does it not cause an issue?